common/traits/wc_race_traits.txt

```diff
@@ -1978,6 +1978,23 @@ creature_ancient_wind = {
 	random = no
 	customizer = no
 }
+creature_ancient_coral = {
```

Comment:
It wasn't added to `is_plant_race_trigger`.
It may be animal, but it's not able to `can_breath_underwater_trigger`, able to `can_go_hunting_trigger`.
Also, it's able to get a class.
It not `is_completely_immortal_trigger`.

Reply:
fixed